# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/check-headers.sh
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/check-headers.sh
--- a/gfx/cairo/cairo/src/check-headers.sh	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/check-headers.sh	Mon May 06 19:25:12 2024 +0000
@@ -11,7 +11,7 @@
 cd "$srcdir"
 FILES=$all_cairo_headers
 if test "x$FILES" = x; then
-	FILES=`find . -name 'cairo*.h' ! -name '*-private.h' ! -name 'cairoint.h'`
+	FILES=`find . -name 'cairo*.h' ! -name '*-private.h' ! -name 'cairoint.h' ! -name '*-inline.h'`
 fi
 
 grep -B 1 '^cairo_.*[ 	]\+(' /dev/null $FILES |